README.md

```diff
@@ -149,7 +155,7 @@ result = Veritrans.create_widget_token(
 
 > !!! WARNING NOTE: VT-Web is deprecated, please use [Snap](#Snap Pop-up) instead, it has better previous VT-Web feature and many more improvements, including redirect_url.
 
-### VT-Direct / Core API
+###Core API (VT-Direct)
 
 It's little more complicated, because credit_card is sensitive data,
 you need put credit card number in our safe storage first using `midtrans.min.js` library, then send received token to with other payment details.
```

Comment:
Fix this, no longer `midtrans.min.js`

Reply:
also improve the wording `It's little more complicated, because credit_card is sensitive data` is not a good one.